.github/workflows/data-host.yml

```diff
@@ -0,0 +1,40 @@
+name: Data Host CI
+
+# Triggers the workflow on pull request events but only for the master branch
+on:
+  pull_request:
+    paths:
+      - 'support/@trustfractal/data-host/**'
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: support/@trustfractal/data-host
+
+jobs:
+  'node-14':
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      # Download and cache dependencies
+      - name: Cache node modules
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: Install packages
+        run: yarn install
+      - name: Run tests
+        run: yarn test
+      #  run: yarn run cov:send
+      #  run: yarn run cov:check
```

Comment:
FWIW, I have no interest in including code coverage. At best, it's a slightly interesting metric, and at worst it makes teams test things they shouldn't and not cover all interesting cases.